plugins/modules/mso_schema_template_external_epg.py

```diff
@@ -35,6 +35,13 @@
     - The name of the external EPG to manage.
     type: str
     aliases: [ name, externalepg ]
+  external_epg_type:
+    description:
+    - The type of external epg.
+    - anp needs to be associated with external epg when the type is cloud.
```

Comment:
add that l3out can be associated with external epg when the type is on-premises.